Regulator_SwitchedCapacitor.lib

```diff
@@ -74,6 +74,29 @@ X C- 9 -400 -600 100 R 50 50 1 1 P
 ENDDRAW
 ENDDEF
 #
+# LM7705
+#
+DEF LM7705 U 0 20 Y Y 1 F N
+F0 "U" -250 350 50 H V C CNN
+F1 "LM7705" 200 350 50 H V C CNN
+F2 "Package_SO:VSSOP-8_3.0x3.0mm_P0.65mm" 0 450 50 H I C CNN
+F3 "" 0 0 50 H I C CNN
+$FPLIST
+ VSSOP*3.0x3.0mm*P0.65mm*
+$ENDFPLIST
+DRAW
+S -300 300 300 -300 0 1 10 f
+X CF+ 1 -400 0 100 R 50 50 1 1 I
+X GND 2 0 -400 100 U 50 50 1 1 W
+X SD 3 -400 200 100 R 50 50 1 1 I
+X VDD 4 0 400 100 D 50 50 1 1 W
+X GND 5 0 -400 100 U 50 50 1 1 P N
+X VOUT 6 400 200 100 L 50 50 1 1 w
+X CRES 7 400 0 100 L 50 50 1 1 I
+X CF- 8 -400 -100 100 R 50 50 1 1 I
```

Comment:
I think that you made a good suggestion: I think that `CRES`, `CF+` and `CF-` should be passive.